src/js/models/menubar.js

```diff
@@ -141,7 +141,7 @@ define(function (require, exports, module) {
             return {
                 "always": true,
                 "always-except-modal": false,
-                "modal-have-document": appIsInputModal && (document !== null)
+                "modal-have-document": appIsInputModal || (document !== null)
```

Comment:
I think this that has one unintended consequence, and that "modal-have-document' is no longer a good name for this prop.

I think what we are now *intending* is "*super search OR have-document*" which currently implies to "*cut/copy/paste/selectAll is enabled*". 

But the logic as of this commit is "super search OR have-document OR (help screens && have-document)".  So, for example, we now allow select-all while looking at the "keyboards shortcut" modal dialog, which has the effect of selecting all layers on the underlying document.  I don't *think* we want that.

I suggested changing this (and associated references to):
`super-search-or-have-document: appIsInputModal`